src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -5,6 +5,37 @@
 from WMCore.WMSpec.WMWorkload import WMWorkloadHelper
 
 
+def convertWQElementsStatusToWFStatus(elementsStatusSet):
```

Comment:
Either I have mental problems, or this is an evil mapping :) It took me quite a bit to digest, I was even wondering if using the "english" methods like `issuperset` wouldn't be better, but I guess it's the same in the end.